apps/air-discount-scheme/backend/migrations/20210311225505-add-marked-flight-column.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(`
+      BEGIN;
+
+        ALTER TABLE flight ADD COLUMN connectable BOOLEAN NOT NULL DEFAULT TRUE;
```

Comment:
I am wondering if storing something like `connectingFlightId` instead of `connectable` would be better.
We could use it the same by checking if it is null and we would also get the connection between the flights stored in the db

Reply:
Yeah it would make sense. Is this something we want to do for phase 2?